docs/getting-started/build-with-spack.md

```diff
@@ -6,6 +6,11 @@ HPC.
 
 To install CP2K with [Spack], you need to [install Spack].
 
+.. note:: 
```

Comment:
Since we're using MyST, the syntax is a bit different:

https://github.com/cp2k/cp2k/blob/master/docs/README.md#notes-and-warnings